test/sqllogictest/tpch.slt

```diff
@@ -760,10 +756,10 @@ ORDER BY
 | |   delta %3 %0.(#0) %1.(#0) %5.(#0) %2.(#1, #2) %4.(#0)
 | |   delta %4 %2.(#0) %3.(#0, #1) %0.(#0) %1.(#0) %5.(#0)
 | |   delta %5 %1.(#3) %2.(#2) %3.(#0, #1) %0.(#0) %4.(#0)
-| | demand = (#1, #20..#22, #35, #41, #47)
-| Filter "^.*green.*$" ~(#1)
-| Reduce group=(#47, date_part_year_tstz(datetotstz(#41)))
-| | agg sum(((#21 * (100dec - #22)) - (#35 * #20)))
+| Project (#1, #20..#22, #35, #41, #47)
+| Filter "^.*green.*$" ~(#0)
```

Comment:
I'm reading through plans without reading the code, but I do see a bunch of this pattern. I think this will be a regression until we implement pushing down MFP execution into joins. At least partially. The code today will push filters in to the join execution, which can be very handy (you can imagine in this case), but this will probably block that.